src/test/java/org/opensearch/neuralsearch/search/HitsTresholdCheckerTests.java

```diff
@@ -27,7 +27,13 @@ public void testTresholdLimit_whenThresholdNegative_thenFail() {
         expectThrows(IllegalArgumentException.class, () -> new HitsThresholdChecker(-1));
     }
 
-    public void testTresholdLimit_whenThresholdMaxValue_thenFail() {
-        expectThrows(IllegalArgumentException.class, () -> new HitsThresholdChecker(Integer.MAX_VALUE));
+    public void testTrackThreshold_whenTrackThresholdSet_thenSuccessful() {
```

Comment:
There is typo in the class name it should be HitsThresholdChecker and not HitsTresholdChecker

Reply:
ack

Reply:
ack